### ISSUE TITLE
Random URL box show on Oxygen editor & color picker doesn't work (Wordpress 5.6 & Oxygen 3.6.1)

### ISSUE THREAD

**Issue by ducminhn** [soflyy/oxygen-bugs-and-features#1787]
I have a very strange issue on my site which has just been updated to WP 5.6 recently. When I go to Oxygen editor to edit any page/template, there's a URL input box randomly show on top of the left panel of the editor (screenshot below). I cannot close this box or do anything with it. It blocks the search box component of Oxygen when I try to find a page to preview the template:

![image](https://user-images.githubusercontent.com/7876935/102722126-e2685680-42b3-11eb-82e4-b4a7bc76b8bd.png)

And this is the error in the console, looks like the jquery-ui library has some conflicts:

![image](https://user-images.githubusercontent.com/7876935/102722150-09bf2380-42b4-11eb-9389-8f9b699c7807.png)

I'm using the BNB design library for my site and this issue happens on all browsers that I can test: Chrome, Firefox & Edge (all in the latest version)

Here is my site info:
```
### wp-core ###

version: 5.6
site_language: vi_VN
user_language: vi_VN
timezone: +00:00
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 0
default_comment_status: open
environment_type: production
user_count: 5
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/wordpress
wordpress_size: 48.28 MB (50626286 bytes)
uploads_path: /var/www/wordpress/wp-content/uploads
uploads_size: 139.25 MB (146013096 bytes)
themes_path: /var/www/wordpress/wp-content/themes
themes_size: 5.04 MB (5289429 bytes)
plugins_path: /var/www/wordpress/wp-content/plugins
plugins_size: 84.83 MB (88947359 bytes)
database_size: 23.48 MB (24625152 bytes)
total_size: 300.89 MB (315501322 bytes)

### wp-active-theme ###

name: Twenty Twenty (twentytwenty)
version: 1.6
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, widgets, menus, post-thumbnails, title-tag, editor-styles, align-wide
theme_path: fake
auto_update: Disabled

### wp-themes-inactive (1) ###

Twenty Twenty-One: version: 1.0, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (16) ###

Admin Menu Editor Pro: version: 2.13, author: Janis Elsts, Auto-updates enabled
Advanced Custom Fields PRO: version: 5.9.3, author: Elliot Condon, Auto-updates enabled
Custom Post Type UI: version: 1.8.1, author: WebDevStudios, Auto-updates enabled
Duplicate Page: version: 4.3, author: mndpsingh287, Auto-updates enabled
Gravity Forms: version: 2.4.21, author: Gravity Forms, Auto-updates enabled
Modern Events Calendar Lite: version: 5.15.0, author: Webnus, Auto-updates enabled
Nested Pages: version: 3.1.13, author: Kyle Phillips, Auto-updates enabled
Oxygen: version: 3.6.1, author: Soflyy, Auto-updates disabled
Oxygen Gutenberg Integration: version: 1.4, author: Soflyy, Auto-updates disabled
Post SMTP: version: 2.0.16, author: Yehuda Hassine, Auto-updates disabled
Simple History: version: 2.38.0, author: Pär Thernström, Auto-updates disabled
Wordfence Security: version: 7.4.14, author: Wordfence, Auto-updates disabled
WP Fastest Cache: version: 0.9.1.3, author: Emre Vona, Auto-updates disabled
WPML Multilingual CMS: version: 4.4.8, author: OnTheGoSystems, Auto-updates enabled
WPML String Translation: version: 3.1.6, author: OnTheGoSystems, Auto-updates disabled
WPML Translation Management: version: 2.10.4, author: OnTheGoSystems, Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
file_uploads: File uploads is turned off
post_max_size: 512M
upload_max_filesize: 512M
max_effective_size: 512 MB
max_file_uploads: 20
gd_version: 2.2.5
ghostscript_version: not available

### wp-server ###

server_architecture: Linux 5.4.60-1-pve x86_64
httpd_software: Apache
php_version: 7.3.19-1~deb10u1 64bit
php_sapi: apache2handler
max_input_variables: 1000
time_limit: 300
memory_limit: 1024M
max_input_time: 60
upload_max_filesize: 512M
php_post_max_size: 512M
curl_version: 7.64.0 OpenSSL/1.1.1d
suhosin: false
imagick_availability: false
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.3.27-MariaDB-0+deb10u1
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: https://stjosephvancouver.ca
WP_SITEURL: https://stjosephvancouver.ca
WP_CONTENT_DIR: /var/www/wordpress/wp-content
WP_PLUGIN_DIR: /var/www/wordpress/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 1024M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

```

**Comment by KittenCodes** [comment]
@ducminhn Please run a plugin conflict test: https://pluginconflict.com/?plugin=Oxygen.

If the issue occurs when Oxygen is the only active plugin on the site, please open a support ticket so that we can take a look: https://oxygenbuilder.com/support.

**Comment by YammYcoding** [comment]
Hey man, I had that very same issue a few days ago. That is caused by HappyFiles plugin beeing outdated, once you update it, the annoying div will get gone!

**Comment by ducminhn** [comment]
Thanks, I found the problem was the conflict with the [Modern Events Calendar Lite](https://wordpress.org/plugins/modern-events-calendar-lite/)
I'm waiting for the update from this plugin to see if it fixes the issue.

**Comment by flore89** [comment]
I need jqueryui on some of my pages. so i have to enqueue it. So on all the pages I enqueue it, it also loads in the oxygenbuilder but _twice_ because it also comes with the builder. thats also where the error comes from.

it its strange that oxygen builder loads the enqueued scripts also in the builder.

how i enqueue the script:

            if ( is_page(array( 59, 40, 39 )) ) {
				wp_enqueue_script('jqueryui', AJ_PLUGIN_URL . 'assets/js/jquery-ui.min.js', array(), $version, true );
				wp_enqueue_script('dwform', AJ_PLUGIN_URL . 'assets/js/form.js', array(), $version, true );

            }

is there something i can put in my if query? something like !is_oxygenbuilder()

**Comment by KittenCodes** [comment]
@flore89 You can use the Code Snippets plugin to prevent the file from being enqueued twice. Please create a snippet with the following code:

```php
add_action( 'wp_enqueue_scripts', function () {

if (! strpos( $_SERVER['REQUEST_URI'], 'ct_builder=true' ) !== false ) return;

foreach ( wp_scripts()->queue as $wp_script ) {
	
	switch( $wp_script ) {
			
		case 'jquery-ui':
			wp_deregister_script( $wp_script );
		break;
			
			
	}
	
}

});
```

**Comment by flore89** [comment]
thank you, works like a charm!

**Comment by KittenCodes** [comment]
This is fixed in Oxygen v3.8 Beta 1.
